permissions/plugin-google-kubernetes-engine.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "google-kubernetes-engine"
+github: "jenkinsci/google-kubernetes-engine-plugin"
+paths:
+- "org/jenkins-ci/plugins/google-kubernetes-engine"
```

Comment:
side note: more and more plugins actually use io.jenkins.plugins nowadays, maybe you'd like to use it too?

@slide what do you think about enriching your checking tool to advise using this groupId for new plugins, instead of the older `org/jenkins-ci/plugins`?

Reply:
What does the plugin creator archetype setup? We should start there :). I can advise people using the checker.